mysql/assets/configuration/spec.yaml

```diff
@@ -361,30 +361,7 @@ files:
                   `performance_schema.global_variables`.
                 value:
                   type: number
-                  example: 600
-          - name: schemas_collection 
```

Comment:
All of the below config files are manually generated so the changes will get over-written. The spec.yaml should have `hidden: true` which will hide the config from the conf.example.yaml.